docker-compose.yml

```diff
@@ -147,16 +147,14 @@ services:
   dataland-edc:
     restart: always
     profiles: ["production"]
-    image: ghcr.io/d-fine/datalandedc/edcserver:0.2.8
+    image: ghcr.io/d-fine/datalandedc/edcserver:0.2.30-SNAPSHOT
     networks:
       - internal
     expose:
       - "9191"
       - "9292"
     environment:
-      DATALAND_EDC_WEB_URI: ${DATALAND_EDC_WEB_URI}
-      DATALAND_EDC_IDS_URI: ${DATALAND_EDC_IDS_URI}
-      TRUSTEE_WEB_URI: ${TRUSTEE_WEB_URI}
+      DATALAND_EDC_ASSET_ACCESS_URI: ${DATALAND_EDC_ASSET_ACCESS_URI}
       TRUSTEE_IDS_URI: ${TRUSTEE_IDS_URI}
```

Comment:
It seems to me that all of these are unused. What was your reason for removing exactly these three? Maybe also remove them in the .env.template.